Formula/avian.rb

```diff
@@ -23,7 +23,7 @@ class Avian < Formula
   def install
     system "make", "use-clang=true"
     bin.install Dir["build/macosx-*/avian*"]
-    lib.install Dir["build/macosx-*/*.dylib", "build/macosx-*/*.a"]
+    lib.install Dir["build/macosx-*/#{shared_library("*")}", "build/macosx-*/*.a"]
```

Comment:
Wouldn't this be in a different directory?

Reply:
I see, you mean in a Linux directory. This is up to discussion. The question will be how I'll teach rubocop to ignore theses cases and not try to enforce shared_library. The cop will basically look at all the strings assigned to variables or arguments passed to functions and ask you to no hardcode .dylib or .so.

Reply:
I think you should just use something like an osname variable in the directory name here. Disclaimer: I don't know if we have one of those.